strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/artifact/locator/handlers/AbstractArtifactLocationHandler.java

```diff
@@ -55,7 +55,7 @@ protected boolean isMetadata(Path p)
     {
         try
         {
-            return Boolean.TRUE.equals(Files.getAttribute(p, RepositoryFileAttributes.METADATA));
+            return Boolean.TRUE.equals(RepositoryFiles.isMetadata((RepositoryPath) p));
```

Comment:
Cosmetic: Could `p` be renamed to `path`, please? Thanks!

Reply:
ok